### PR TITLE
add bento.me

### PR DESCRIPTION
**Pull request by Its-Just-Nans**

### CODE REVIEW

File: wmn-data.json [hunk 1]
```diff
@@ -6432,6 +6432,16 @@
         "m_code" : 404,
         "known" : ["carlito", "taylor"],
         "cat" : "social"
+      },
+      {
+        "name" : "Bento",
+        "uri_check" : "https://bento.me/{account}",
+        "e_code" : 200,
+        "e_string" : "Create Your Bento",
```
Comment (WebBreacher) [inline wmn-data.json:6440]: Please consider changing this to something more unique like `createdAt\":`

File: wmn-data.json [hunk 2]
```diff
@@ -6432,6 +6432,16 @@
         "m_code" : 404,
         "known" : ["carlito", "taylor"],
         "cat" : "social"
+      },
+      {
+        "name" : "Bento",
+        "uri_check" : "https://bento.me/{account}",
+        "e_code" : 200,
+        "e_string" : "Create Your Bento",
+        "m_string" : "Claim",
```
Comment (WebBreacher) [inline wmn-data.json:6441]: Let's also change this to something more unique like `>Available!</div>`

File: wmn-data.json (hunk 1 above)
Comment (Its-Just-Nans) [reply wmn-data.json]: Updated to

```
      {
        "name" : "Bento",
        "uri_check" : "https://bento.me/{account}",
        "e_code" : 200,
        "e_string" : "href=\"https://bento.me/explore\">",
        "m_string" : ">Available!</div>",
        "m_code" : 404,
        "known" : ["carlito", "taylor"],
        "cat" : "social"
      }
```

File: wmn-data.json (hunk 2 above)
Comment (Its-Just-Nans) [reply wmn-data.json]: Updated to

```
      {
        "name" : "Bento",
        "uri_check" : "https://bento.me/{account}",
        "e_code" : 200,
        "e_string" : "href=\"https://bento.me/explore\">",
        "m_string" : ">Available!</div>",
        "m_code" : 404,
        "known" : ["carlito", "taylor"],
        "cat" : "social"
      }
```

File: wmn-data.json [hunk 3]
```diff
@@ -6437,8 +6437,8 @@
         "name" : "Bento",
         "uri_check" : "https://bento.me/{account}",
         "e_code" : 200,
-        "e_string" : "Create Your Bento",
-        "m_string" : "Claim",
+        "e_string" : "href=\"https://bento.me/explore\">",
```
Comment (WebBreacher) [inline wmn-data.json:6440]: Please check this string. In my testing, I don't see the ending `>`.

Comment (Its-Just-Nans) [reply wmn-data.json]: true

**Comment by WebBreacher** [comment]
One more change. Entries should be alphabetized by their name "Bento" instead of being placed at the end of the JSON file. Please move this entry into the "B" names.

**Comment by Its-Just-Nans** [comment]
okay
i fixed npm also then

**Comment by WebBreacher** [comment]
Thank you!

**Pull request merged**